### PR TITLE
Remove variables from metrics-exporter

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

Depends-On: https://github.com/thoth-station/metrics-exporter/issues/426

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Remove variables for metrics-exporter

### CODE REVIEW

File: metrics-exporter/base/deploymentconfig.yaml [hunk 1]
```diff
@@ -168,21 +168,6 @@ spec:
                 configMapKeyRef:
                   name: prometheus
                   key: instance-metrics-exporter-frontend
-            - name: WORKFLOW_METRICS_BACKEND_PROMETHEUS_INSTANCE
```
Comment (goern) [inline metrics-exporter/base/deploymentconfig.yaml:171]: could you remove that form the configmap in core too?

Comment (pacospace) [reply metrics-exporter/base/deploymentconfig.yaml:171]: Done :)

**Pull request merged**